tt_metal/hostdevcommon/common_runtime_address_map.h

```diff
@@ -55,29 +55,19 @@ static_assert (PROFILER_FULL_HOST_BUFFER_SIZE_PER_RISC > PROFILER_L1_BUFFER_SIZE
 // Size is presently based on the old size of the RTAs (large enough to hold 1 set)
 // plus some extra space freed up in the mem map
 constexpr static std::uint32_t L1_KERNEL_CONFIG_BASE = PROFILER_L1_END_ADDRESS;
-constexpr static std::uint32_t L1_KERNEL_CONFIG_SIZE = 4 * 1024 + 256;
+constexpr static std::uint32_t L1_KERNEL_CONFIG_SIZE = 4 * 1024 + 256 + 128 + 512;
```

Comment:
Can you update the comment above to specify that cbs and semaphores were added and the constant calculations for the respective sizes